### PR TITLE
Added next-source-maps plugin

### PR DESCRIPTION
**Pull request by ptomasroos**

### CODE REVIEW

File: packages/next-source-maps/package.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "name": "@zeit/next-source-maps",
```
Comment (timneutkens) [inline packages/next-source-maps/package.json]: Should it be called `@zeit/next-production-source-maps` 🤔

Comment (ptomasroos) [reply packages/next-source-maps/package.json]: Nope

**Comment by timneutkens** [comment]
Published @zeit/next-source-maps@0.0.1

**Comment by kachkaev** [comment]
Should it be also added to README.md?

**Comment by timneutkens** [comment]
@kachkaev just added it 👍

**Pull request merged**